tests/unit-tests/api/utilities/Tests_Functions.php

```diff
@@ -0,0 +1,497 @@
+<?php
```

Comment:
[This comment](https://github.com/Getbeans/Beans/pull/67#discussion_r151621237) applies to this file too.